signerverifier/ecdsa.go

```diff
@@ -0,0 +1,94 @@
+package signerverifier
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/sha256"
+	"crypto/sha512"
+	"os"
+)
+
+const ECDSAKeyType = "ecdsa"
+
+type ECDSASignerVerifier struct {
+	keyID     string
+	curveSize int
+	private   *ecdsa.PrivateKey
+	public    *ecdsa.PublicKey
+}
+
+func NewECDSASignerVerifierFromSSLibKey(key *SSLibKey) (*ECDSASignerVerifier, error) {
+	_, publicParsedKey, err := decodeAndParsePEM([]byte(key.KeyVal.Public))
+	if err != nil {
+		return nil, err
```

Comment:
nit: a best practice is to use `fmt.Errorf` when returning an error with a description.